docs/static/source/data_index.js

```diff
@@ -344,10 +350,13 @@ indexData = [
   ["ComObjValue()","commands/ComObjValue.htm",2],
   ["Compatibility","Compat.htm"],
   ["compile a script","Scripts.htm#ahk2exe"],
+  ["Compiler Drectives","Ahk2ExeDirectives.htm"],
```

Comment:
Typo "Drectives"

Reply:
Thanks for reviewing :)
I can fix this if you like. I also want to add 'BinMod' to the indexes, so I’ll do that at the same time.
Let me know if you find anything else.
Regards